### ISSUE TITLE
Sonar coverage reporter shows absolute path files

### ISSUE THREAD

**Issue by jgarec** [utPLSQL/utPLSQL-maven-plugin#10]
Hi,

Is there any reason why findSQLs adds absolute file names instead of relative path names ?

https://github.com/utPLSQL/utPLSQL-maven-plugin/blob/0946db122f45ff29fd6072b05e5e66c756e005bf/utplsql-maven-plugin/src/main/java/org/utplsql/helper/SQLScannerHelper.java#L38

I think that the following code should be ok :
```
founds.add(new File(scanner.getBasedir(), basename).getPath());
```

**Comment by alberto-hernandez** [comment]
Fixed.

**Comment by jgarec** [comment]
#10 fixes this issue and #15 adds tests
